javascript2/week1/script.js

```diff
@@ -0,0 +1,123 @@
+let recipeObject = {
+  id: 1,
+  title: "Vegetable Biryani",
+  picture_url: "./images/veg_biryani.jpeg",
+  ingredients: [
+    { NAME: "bay leaf", AMOUNT: "2" },
```

Comment:
Minor: The assignment description had these in UPPER_CASE, but there's probably no good reason for that - I would recommend using the normal camelCase convention for property names.